omniledger/service/service.go

```diff
@@ -1193,5 +1224,26 @@ func newService(c *onet.Context) (onet.Service, error) {
 		return nil, err
 	}
 	s.skService().RegisterStoreSkipblockCallback(s.updateCollectionCallback)
+	s.skService().EnableViewChange()
+
+	// Register the view-change cosi protocols.
+	var err error
+	_, err = s.ProtocolRegister(viewChangeSubFtCosi, func(n *onet.TreeNodeInstance) (onet.ProtocolInstance, error) {
+		return cosiprotocol.NewSubFtCosi(n, s.verifyViewChange, cothority.Suite)
+	})
+	if err != nil {
+		return nil, err
+	}
+	_, err = s.ProtocolRegister(viewChangeFtCosi, func(n *onet.TreeNodeInstance) (onet.ProtocolInstance, error) {
+		return cosiprotocol.NewFtCosi(n, s.verifyViewChange, viewChangeSubFtCosi, cothority.Suite)
+	})
+	if err != nil {
+		return nil, err
+	}
+
+	if err := s.tryLoad(); err != nil {
+		log.Error(err)
```

Comment:
99% sure that onet will log this (actually panic this I think), so no need here.